src/System.Drawing.Graphics/src/System/Drawing/Graphics/Image.cs

```diff
@@ -76,6 +76,11 @@ private Image(Stream stream)
             throw new NotImplementedException();
         }
 
+            /* Manipulations */
+        public Image Write(string im, string dst)
```

Comment:
these parameter names don't follow our naming conventions. They need to not use abbreviations.